i18n/translation-ko.json

```diff
@@ -799,7 +799,7 @@
     "cropDimensions": "크기",
     "dimensionInput": {
       "unitOfMeasurement": "단위",
-      "autoTrim": "“자동"
+      "autoTrim": "자동트림"
```

Comment:
Based on context we have the following suggestion:
```
“autoTrim”: “자동트림” => “자동 자르기”  or “자동”
```

![image](https://user-images.githubusercontent.com/10825527/175137402-6ad701dc-508c-4bb7-9b84-9c234859c338.png)